lms/static/sass/views/_teams.scss

```diff
@@ -148,6 +149,200 @@
     @extend %t-weight4;
   }
 
+  .search-tools {
+    @extend %t-copy-sub1;
+    margin: ($baseline/10) $baseline;
+    color: $gray-l1;
+
+    p, span { // override
+      color: inherit;
+    }
+  }
+
+  .card {
+    @extend .clearfix;
+    position: relative;
+    display: inline-block;
+    width: 30%;
+    margin: ($baseline) 1.2%;
```

Comment:
Hmm, I know you've had to compensate like this in some previous PRs when building micro-layouts. A quick comment with some sort of grep-able label would be good to start. Something like ``GRID: ___`` or ``LAYOUT: ___`` might make sense.